espnet2/asr/postencoder/transformers.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+#  2021, University of Stuttgart;  Pavel Denisov
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+"""Hugging Face Transformers PostEncoder."""
+
+from espnet.nets.pytorch_backend.nets_utils import make_pad_mask
+from espnet2.asr.postencoder.abs_postencoder import AbsPostEncoder
+from typeguard import check_argument_types
+from typing import Tuple
+
+import torch
+
+
+class Transformers(AbsPostEncoder):
+    """Hugging Face Transformers PostEncoder."""
+
+    def __init__(
+        self,
+        input_size: int,
+        model_name_or_path: str,
+    ):
+        """Initialize the module."""
+        assert check_argument_types()
+        super().__init__()
+
+        try:
+            from transformers import AutoModel
+        except Exception as e:
+            print("Error: Transformers is not properly installed.")
+            print(
+                "Please install Transformers: cd ${MAIN_ROOT}/tools && make transformers.done"
+            )
+            raise e
+
+        model = AutoModel.from_pretrained(model_name_or_path)
+        self.transformer = model.encoder
```

Comment:
Did you make sure that the parameters of `self.transformer` wouldn't be messed up by the [initialization](https://github.com/espnet/espnet/blob/2a9e8ad0b95fe0bababa88dec53fc8ce6ba55059/espnet2/tasks/asr.py#L456)?

Reply:
They were, fixed now.